.github/ISSUE_TEMPLATE/bug_v3.yaml

```diff
@@ -0,0 +1,94 @@
+name: 🐛 Bug Report v3
+description: Use this template to report bugs for version 3.
+title: "🐛 | "
+labels: ["bug", "future3", "needs triage"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Please fill out the form to provide essential information to your problem.
+        This will help us to faster figure out the root cause.
+
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: |
+        What version do you use?
+        See the "settings" page in the Web App or the "src/jukebox/jukebox/version.py" file.
+      placeholder: e.g. "3.5.0" or "3.6.0-alpha"
+    validations:
+      required: true
+
+  - type: input
+    id: branch
+    attributes:
+      label: Branch
+      description: |
+        What branch did you install from?
+        See the "settings" page in the Web App or the "src/jukebox/jukebox/version.py" file.
+      placeholder: e.g. "main" or "develop"
+    validations:
+      required: true
+
+  - type: input
+    id: os
+    attributes:
+      label: OS
+      description: |
+        What Operation System and Version do you use?
+        See `cat /etc/os-release`
+      placeholder: e.g. "Raspberry Pi OS Bullseye lite - 32bit"
+    validations:
+      required: true
+
+  - type: input
+    id: pi-model
+    attributes:
+      label: Pi model
+      description: |
+        Which Raspberry Pi model do you use?
+      placeholder: e.g. "3 B+", "Zero 2"
+    validations:
+      required: true
+
+  - type: textarea
+    id: hardware
+    attributes:
+      label: Hardware
+      description: |
+        What's your hardware set up?
+        e.g. Rfid Reader, SoundCards, ...
+
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: |
+        What you did, what you expected and what happened instead?
+    validations:
+      required: true
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: Logs
+      description: |
+        Please provide the logs after the problem occured.
```

Comment:
Add web interface at `http://ip.of.your.box/logs`  as well. Could be easier for some users.